zcash_client_backend/src/keys.rs

```diff
@@ -283,6 +294,11 @@ impl UnifiedFullViewingKey {
         self.sapling.as_ref()
     }
 
+    /// Returns the Orchard full viewing key component of this unified key.
```

Comment:
```suggestion
    /// Returns the Orchard full viewing key component of this unified key.
    /// (Orchard full viewing keys are always diversifiable.)
```

Reply:
I don't understand what it means for a full viewing key to be diversifiable?

Reply:
I think @daira was contrasting this to the docstring for the Sapling side, which says it returns a "diversifiable full viewing key". I think a better resolution would be to remove "diversifiable" from the Sapling text, as it's more of an implementation detail that users shouldn't need to care about.